extension/src/extension.ts

```diff
@@ -63,6 +66,20 @@ export function activate(context: vscode.ExtensionContext) {
         });
     });
 
+    const disposableBuild = vscode.commands.registerCommand("xcomponent.build.project", () => {
+        const rootPath = vscode.workspace.rootPath;
+        const baseName = path.basename(rootPath);
+        const xcmlPath = `${rootPath}${path.sep}${baseName}_Model.xcml`;
+        const buildCommand = `xcbuild --compilationmode=Debug --build --env=Dev --vs=VS2015 --project=${xcmlPath}`;
```

Comment:
this command works only on windows.
other systems use mono: mono xcbuild.exe

Reply:
It could be interesting to provide a setting to customize xcbuild path (the same way as the python path in this extension [https://github.com/DonJayamanne/pythonVSCode/wiki/Python-Path-and-Version#manual-configuration](https://github.com/DonJayamanne/pythonVSCode/wiki/Python-Path-and-Version#manual-configuration) )